lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -4076,8 +4079,17 @@ static void fwdArgMax(Tensor *argmaxW, Tensor *inW, size_t axis) {
   for (a = 0; a < idim[(axis + 1) % 4]; a++) {
     for (b = 0; b < idim[(axis + 2) % 4]; b++) {
       for (c = 0; c < idim[(axis + 3) % 4]; c++) {
+        T max;
+        if (axis == 0) {
+          max = inH.at({0, *dim[1], *dim[2], *dim[3]});
+        } else if (axis == 1) {
+          max = inH.at({*dim[0], 0, *dim[2], *dim[3]});
+        } else if (axis == 2) {
+          max = inH.at({*dim[0], *dim[1], 0, *dim[3]});
+        } else {
+          max = inH.at({*dim[0], *dim[1], *dim[2], 0});
+        }
```

Comment:
nit: This works/I'm fine with it -- but it would probably be simpler to just set `T max = std::numeric_limits<T>::min()`.

Reply:
Done.